src/library/scala/collection/convert/DecorateAsJava.scala

```diff
@@ -18,91 +18,91 @@ import scala.language.implicitConversions
 trait DecorateAsJava extends AsJavaConverters {
   /**
    * Adds an `asJava` method that implicitly converts a Scala `Iterator` to a Java `Iterator`.
```

Comment:
What's the policy on updating the copyright header? As you go or bulk update later>